src/main/scala/beam/physsim/jdeqsim/JDEQSimRunner.scala

```diff
@@ -345,7 +350,7 @@ object JDEQSimRunner {
               val ftt = link.getLength / link.getFreespeed(time)
               if (volume >= minVolumeToUseBPRFunction) {
                 val tmp = volume / (link.getCapacity(time) * flowCapacityFactor)
-                val originalTravelTime = ftt * (1 + tmp * tmp)
+                val originalTravelTime = ftt * ftt * (1 + tmp * tmp)
```

Comment:
I think you want to do the same thing down here that you did up there

Reply:
fixed, thanks!